### PR TITLE
Combine

### PR DESCRIPTION
**Pull request by andraztori**
- fix a small bug with not accepting "w" as a valid character for a namespace
- allow for transformations of transformed namespaces
- implement Combine transformer

### CODE REVIEW

**Comment by andraztori** [comment]
@adischw  anything new here :)

File: BENCHMARK.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ or, if you just want the numbers with less dependencies run:
 ### versions:
 ```
 vowpal wabbit 8.9.2 (git commit: 884420267)
-fwumious wabbit 1.3 (git commit: a98c1e6)
+fwumious wabbit 0.1 (git commit: b89fc4f)
```
Comment (adischw) [inline BENCHMARK.md:85]: shouldnt it be 1.4?

**Review by adischw — APPROVED** [review]

File: src/version.rs [hunk 2]
```diff
@@ -1 +1 @@
-pub static LATEST: &str = "1.3";
+pub static LATEST: &str = "0.1";
```
Comment (adischw) [inline src/version.rs:1]: 1.4?

Comment (andraztori) [reply src/version.rs:1]: dont know how this is supposed to work, honestly :) please fix it if needed

**Comment by andraztori** [comment]
Also added:
- implement Moran Gavish's idea about substracting greater than before applying ln
- change order of arguments of Binners to match LogRatioBinner (first greater than, then resolution)
- split actual implementations into its own file
- drop use of macros to create seeds (there was no point in the first place)
- reduce number of parameters combine can handle to 4 (from 5), as compile times were insane

**Pull request merged**